htdocs/admin/multicurrency.php

```diff
@@ -0,0 +1,289 @@
+<?php
+/* <one line to give the program's name and a brief idea of what it does.>
+ * Copyright (C) 2015 ATM Consulting <support@atm-consulting.fr>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * 	\file		admin/multicurrency.php
+ * 	\ingroup	quickpriceupdate
+ * 	\brief		This file is an example module setup page
+ * 				Put some comments here
+ */
+// Dolibarr environment
+
+require '../main.inc.php';
+
+// Libraries
+dol_include_once('/core/lib/admin.lib.php');
```

Comment:
:+1:  All right, if this feature is merge I'll work again to fix it.  Actually i'm waiting for good return about this PR to integrate the rest.